docs/src/install/index.md

```diff
@@ -180,6 +180,19 @@ curl -fsSL https://thin-edge.io/install.sh | sh -s -- --package-manager tarball
 wget -O - https://thin-edge.io/install.sh | sh -s -- --package-manager tarball
 ```
 
+## Community plugins
+
+In addition to the core thin-edge packages,
+there are several community plugins maintained and developed by the thin-edge open source community.
+These plugins provide additional functionality and integrations for thin-edge.io, enabling users to extend
+the capabilities of their IoT solutions.
+Some of the popular community plugins are listed [here](../operate/plugins/index.md).
+
+:::note
+The community plugins maintained by thin-edge are hosted in a separate [community repository on Cloudsmith](https://cloudsmith.io/~thinedge/repos/community/packages).
+Instructions to setup this repo are documented [here](https://cloudsmith.io/~thinedge/repos/community/setup).
+:::
```

Comment:
You should use `%%te%%` when referring to thin-edge.io. This is a special variable which will be replaced with "thin-edge.io" when the documents are rendered.

Reply:
Resolved by 7cc7d52599dcc6111846a41a4698a449427f51dc